pkg/innerring/processors/reputation/process_put.go

```diff
@@ -48,3 +61,18 @@ func (rp *Processor) processPut(epoch uint64, id reputation.PeerID, value reputa
 			zap.String("error", err.Error()))
 	}
 }
+
+func (rp *Processor) checkManagers(e uint64, mng apireputation.PeerID, peer apireputation.PeerID) error {
+	mm, err := rp.mngBuilder.BuildManagers(e, reputation.PeerIDFromBytes(peer.ToV2().GetPublicKey()))
+	if err != nil {
+		return fmt.Errorf("could not build managers: %w", err)
+	}
+
+	for _, m := range mm {
+		if bytes.Equal(mng.ToV2().GetPublicKey(), m.PublicKey()) {
```

Comment:
We don't want `ToV2`, so at least add FIXME.

Reply:
added `FIXME` to every `ToV2` usage in that package